integrationTests/frontend/wallet/txInterception_test.go

```diff
@@ -100,32 +98,16 @@ func testInterceptedTxFromFrontendGeneratedParams(
 		t.Skip("this is not a short test")
 	}
 
-	dPool := createTestDataPool()
-	startingNonce := uint64(0)
-
-	addrConverter, _ := addressConverters.NewPlainAddressConverter(32, "0x")
-	accntAdapter := createAccountsDB()
-
-	shardCoordinator := &sharding.OneShardCoordinator{}
-
-	n, _, sk, _ := createNetNode(dPool, accntAdapter, shardCoordinator)
-
-	//set the account's nonce to startingNonce
-	nodePubKeyBytes, _ := sk.GeneratePublic().ToByteArray()
```

Comment:
You could add 
```go
// SetAccountNonce sets the account nonce with journal
func (tpn *TestProcessorNode) SetAccountNonce(nonce uint64) {
	nodePubKeyBytes, _ := tpn.SkTxSign.GeneratePublic().ToByteArray()
	nodeAddress := CreateAddresFromAddrBytes(nodePubKeyBytes)
	nodeAccount, _ := tpn.AccntState.GetAccountWithJournal(nodeAddress)
	_ = nodeAccount.(*state.Account).SetNonceWithJournal(nonce)
	_, _ = tpn.AccntState.Commit()
}
```
 to the TestProcessorNode, and use the method here

Reply:
done. also treated errors